app.js

```diff
@@ -452,18 +458,41 @@ window.addEventListener("keydown", e => {
 
 
 let last_scroll = document.getElementById('scroll').scrollTop
-document.getElementById('scroll').addEventListener("scroll", e => {
-	let new_scroll = document.getElementById('scroll').scrollTop
+document.getElementById('scroll').addEventListener('scroll', e => {
+	const { scrollTop, scrollHeight, clientHeight } = document.getElementById('scroll')
+
 	if (is_initial_scroll) {
-		last_scroll = new_scroll
-		setTimeout(() => {is_initial_scroll = false}, 500)		// smooth scrolling can be still going on
+		last_scroll = scrollTop
+		setTimeout(() => {
+			is_initial_scroll = false		// smooth scrolling can be still going on
+			setInterval(() => {
+				let scroll = document.getElementById('scroll')
+				if(scroll.scrollTop == 0) scroll.scroll(0,2)
```

Comment:
keby sa ti chcelo ujednotit pouzivanie `==` nech mame vsade `===` a `!==` tak mozes :D

Reply:
tu to môžem zmeniť ale celkovo chystám to prejsť a nejak ujednotiť. Tak som si povedal, že takéto veci budem riešiť až potom vo "formátovacom pull requeste"